src/librustc_metadata/locator.rs

```diff
@@ -845,6 +860,15 @@ fn get_metadata_section_imp(target: &Target,
                 Ok(blob)
             }
         };
+    } else if flavor == CrateFlavor::Rmeta {
+        let mut file = File::open(filename).map_err(|_|
+            format!("could not open file: '{}'", filename.display()))?;
```

Comment:
Oh ok, I guess whatever is fine. We shouldn't ever throw away the error context, but it's not really that critical.